cmd/action/version.go

```diff
@@ -0,0 +1,50 @@
+package action
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	"ariga.io/atlas/cmd/action/internal/build"
+	"github.com/spf13/cobra"
+)
+
+var (
+	versionCmd = &cobra.Command{
+		Use:    "version",
+		Hidden: true,
+		Run:    cmdVersionRun,
+	}
+)
+
+func init() {
+	RootCmd.AddCommand(versionCmd)
+	RootCmd.Version = build.Version
+
+}
+
+func cmdVersionRun(cmd *cobra.Command, args []string) {
+	fmt.Println(format(build.Version, build.Date))
```

Comment:
you usually like using the cobra std out instead of the std lib :-)

Reply:
Fixed